core/modules/field/css/field.css

```diff
@@ -29,6 +29,7 @@ form .field-multiple-table .field-multiple-drag .tabledrag-handle {
 [dir="rtl"] form .field-multiple-table .field-multiple-drag .tabledrag-handle {
   padding-left: .5em;
 }
-form .field-add-more-submit {
-  margin: .5em 0 0;
+form input.field-add-more-submit {
+  margin: 0;
+  margin-bottom: 0;
```

Comment:
Stumbled by accident on this. In the sandbox, the `margin` is enough, `margin-bottom` doesn't seem necessary to me.

Reply:
The `margin-bottom` was added because `input.form-submit` has a `margin-bottom: 1em;` that does not get overridden by the `margin: 0;`.

Reply:
I just tested this in the sandbox, and removing the `margin-bottom` didn't seem to change anything -- even with CSS aggregation disabled. But I added it because I was still seeing the bottom margin after adding `margin: 0;`. Maybe something else changed since then, and it's not an issue anymore. I'll push an update to the PR and we can test again :)

Reply:
Thanks for the update! Tested in Chrome, looks good to me, the `margin-bottom` gets overridden by the `margin: 0`, both in the inspector (see screenshot) and on the page.

<img width="325" alt="backdrop-add-another-submit-margin" src="https://user-images.githubusercontent.com/14188307/133942910-f442e217-8946-4866-8e0c-5087adc6c3b9.png">